translations/strings.yaml

```diff
@@ -386,4 +386,5 @@ mobileOnboarding:
   signUpBtn: Sign up
   alreadyASubscriber: Already a subscriber?
 
-
+inAppAuth:
+  changeEmail: Change email
```

Comment:
Can we add the role in the ID, e.g. button or link?

Reply:
EDIT: given the screenshot, `changeEmailLink` should work.

Reply:
done